backend/db/schema.py

```diff
@@ -0,0 +1,25 @@
+from sqlalchemy.orm import DeclarativeBase, Mapped, mapped_column
+from datetime import datetime, timezone
+
+
+def generate_utc_timestamp():
+    return datetime.now(timezone.utc)
+
+
+class Base(DeclarativeBase):
+    created_at: Mapped[datetime] = mapped_column(
+        default=generate_utc_timestamp
+    )
+    updated_at: Mapped[datetime] = mapped_column(
+        default=generate_utc_timestamp,
+        onupdate=generate_utc_timestamp
+    )
+
+
+class UserAccount(Base):
```

Comment:
Sorry -- one last request on this schema :joy: . When talking with @MReyna12 , we realized it may be a nice sanity check for ourselves to add a unique constraint on the `email` column for this table. Can we add that real quick?

Reply:
You can blame me 😆.